### ISSUE TITLE
shuffle training set

### ISSUE THREAD

**Issue by feiyu-chen96** [feiyu-chen96/feiyu-chen96.github.io#1]
Shuffling training set (with numpy.random.shuffle or torch.permute) results in a pytorch error.

**Comment by feiyu-chen96** [comment]
oops, it should not be in this repo...
